.travis.yml

```diff
@@ -13,8 +13,9 @@ script:
   - make test
   - bundle exec codeclimate-test-reporter
 env:
-  - RACK_ENV=test
-  - RAILS_ENV=test
+- RACK_ENV=test
+- RAILS_ENV=test
+- COVERAGE=true
```

Comment:
I am pretty confident this is the case because after this PR the two builds for `RACK_ENV` and `RAILS_ENV` failed because they did not have `COVERAGE` set